src/lib/template_style/_form.scss

```diff
@@ -27,120 +35,156 @@
   }
 }
 
-input[type='checkbox'],
-input[type='radio'] {
-  // We don't hide the input tag with display: none because it makes the input inaccessible (you can't 'tab' into it)
-  // See https://www.a11ywithlindsey.com/blog/create-custom-keyboard-accesible-checkboxes
-  position: absolute !important;
-  height: 1px;
-  width: 1px;
-  overflow: hidden;
-  clip: rect(1px, 1px, 1px, 1px);
-  margin-left: unset;
-}
+@mixin customCheckbox($checkbox-top-offset: 0px) {
```

Comment:
All of the custom checkbox/radio styles now live in this mixin. Absolute positioning offsets are relative to each other. The total offset can be overridden by embedding this mixin with a custom parameter.